api_specifications/application-api-0.1.0.yaml

```diff
@@ -310,6 +310,21 @@ components:
           example: Application 1 of the year 2024
         issuedBy:
           $ref: '#/components/schemas/IssuedBy'
+        applicationRaisedBy:
+          type: string
```

Comment:
Why do I need this? There is already a field - onBehalfOf

Reply:
@atulgupta2024  onBehalfOf is the ID of the users for whom the application has been raised. Application Raised by is the Party Name of the advocate raising application. Is this two same?